jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-view.js

```diff
@@ -112,6 +120,8 @@ define([
                     if ($helpArea) {
                         // remove if it exists since layout-specific help text will be inserted
                         $helpArea.remove();
```

Comment:
remove() appears to leaks descendent handlers.

https://jsfiddle.net/obL75u7k/4/

![screen shot 2016-05-11 at 2 11 59 pm](https://cloud.githubusercontent.com/assets/153745/15191471/dbdc5e4e-1782-11e6-8f36-13c6e6dabaa4.png)

Yellow is number of registered event handlers. Up, up up!